vendor/css/custom_bootstrap.css

```diff
@@ -0,0 +1,103 @@
+/* This is custom bootstrap */
+
+.container{
```

Comment:
too much abstartiction , it is better to use eg. header-contianer  , card-container